ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -156,11 +144,17 @@ private void verifyTableIdHasNotChanged(CompactionInfo ci, Table originalTable)
   public void init(AtomicBoolean stop) throws Exception {
     super.init(stop);
 
-    StringBuilder name = new StringBuilder(hostname());
+    StringBuilder name = new StringBuilder(ServerUtils.hostname());
```

Comment:
Could you extract this to getWorkerId()?

Reply:
done